app/templates/dashboard/index.hbs

```diff
@@ -0,0 +1,7 @@
+{{partial 'offers/nav_bar'}}
```

Comment:
how come `index.hbs` and `my_offers.hbs` have the exact same code ? 🤔

Reply:
These two are separate url both have same UI so created partials and shared amongst them.